lib/local-storage-adapter.js

```diff
@@ -20,7 +19,7 @@ function createLocalStorageAdapter (queueName) {
   var setPending = false
   var queueCache = []
 
-  var flushOnDefer = _.debounce(flush, 0)
+  var flushOnDefer = onceOnDefer(flush)
```

Comment:
the `once`ing is the only real functional change. Isn't this a little significant as `flushOnDefer` gets called when updating the queue? Why the once?